apiary.apib

```diff
@@ -250,14 +250,25 @@ or set account properties or destroy their accounts.
 
             Accept: application/vnd.api+json
 
+    + Attributes (object)
+        + data (object, required)
+            + type (string, required)
+            + attributes (object, required)
+                + username (string, required)
+                + password (string, required)
+                + createdAt (string, optional) - timestamp the user account was created on the client
```

Comment:
I think I would leave out the `createdAt` attribute here, and set username & password to optional. They are just examples, the app implementing the spec can decide what attributes they want to use for their accounts.